src/backend/aspen/database/models/mixins.py

```diff
@@ -1,19 +1,58 @@
 import datetime
 import enum
+from typing import Any, Dict, List, NoReturn, Optional, Tuple, Type, TypeVar
 
+import sqlalchemy as sa
 from sqlalchemy import Column, Integer
 
+from aspen.api.deps import get_db
```

Comment:
Makes me kind of nervous to have code that the Flask app depends be dependent on the FastAPI side of things. The `get_db` call doesn't actually do anything with FastAPI **and** it looks like it only gets used in async stuff that I assume won't be used by Flask. But just `get_db` coming from that side of the divide makes me worried. If we see issues during the switchover and need to quickly revert to a pre-FastAPI integration I'm worried about this getting hairy. That said, I don't know if it's really worth extracting those deps out of the FastAPI app folder and into somewhere else...

Reply:
Yeah it's not ideal, but it's just an import dependency -- I think the risk of problems is fairly low here.

Reply:
@jgadling todo -- doublecheck cli/db.py works with this add.